packages/react/src/hooks/useTrackByName.ts

```diff
@@ -0,0 +1,16 @@
+import { useEnsureTrackRef } from '../context';
+import type { UseMediaTrackOptions } from './useMediaTrack';
+import { useMediaTrackBySourceOrName } from './useMediaTrackBySourceOrName';
+import type { TrackReferenceOrPlaceholder } from '@livekit/components-core';
+
+/**
+ * @public
+ * @deprecated this function will be removed in a future version, use `useTrackByName` instead
```

Comment:
Why would you want to add it and deprecate it in the same commit?

Reply:
yeah, that was copy+paste...